src/layouts/components/Editor/extensions/Cards/Cards.ts

```diff
@@ -161,6 +163,22 @@ export const IsomerCards = Node.create<CardOptions>({
         editor.state.selection.replace(tr)
         return true
       },
+      setCardsContent: (content) => ({ tr, dispatch, editor }) => {
+        if (dispatch) {
+          const offset = tr.selection.anchor
+          const node = createCardGridWithContent(editor.schema, content)
+
+          tr.replaceSelectionWith(node)
+            .scrollIntoView()
+            .setSelection(
+              Selection.near(
+                tr.doc.resolve(tr.doc.resolve(offset).parentOffset)
+              )
+            )
+        }
+
+        return true
```

Comment:
sanity check: Is there any semantic meaning to returning true? 
do we need to return false for any case?

Reply:
The true and false is to indicate to Tiptap on whether the command has successfully completed or not. To copy from their docs:

> Both calls would return true if it’s possible to apply the commands, and false in case it’s not.
> 
> In order to make that work with your custom commands, don’t forget to return true or false.

Docs link: https://tiptap.dev/api/commands#dry-run-for-commands